packages/devextreme-scss/scss/widgets/base/chat/layout/chat-messagebubble/_index.scss

```diff
@@ -7,6 +7,14 @@
   width: auto;
   word-break: break-word;
   white-space: break-spaces;
+
+  .dx-chat-messagebubble-content {
+    &.dx-chat-messagebubble-deleted {
+      display: flex;
+      align-items: flex-end;
```

Comment:
why flex-end is here? it will broke appearance when text moved to second line

<img width="252" alt="image" src="https://github.com/user-attachments/assets/30dd7845-1757-4f4e-8204-7eb2252801bc" />